### PR TITLE
Simplify server availability 

### PR DESCRIPTION
**Pull request by paulzierep**
Only get counts for available STAR servers

### CODE REVIEW

File: bin/extract_galaxy_tools.py [hunk 1]
```diff
@@ -27,12 +27,12 @@
 BIOTOOLS_API_URL = "https://bio.tools"
 # BIOTOOLS_API_URL = "https://130.226.25.21"
 
-GALAXY_SERVER_URLS = [
-    "https://usegalaxy.org",
-    "https://usegalaxy.org.au",
-    "https://usegalaxy.eu",
-    "https://usegalaxy.fr",
-]
+USEGALAXY_STAR_SERVER_URLS = {
```
Comment (bebatut) [inline bin/extract_galaxy_tools.py:30]:
```suggestion
USEGALAXY_SERVER_URLS = {
```

File: bin/extract_galaxy_tools.py [hunk 2]
```diff
@@ -761,11 +724,17 @@ def reduce_ontology_terms(terms: List, ontology: Any) -> List:
...
         for tool in tools:
+
+            # add EDAM terms without superclass
             tool["EDAM operation (no superclasses)"] = reduce_ontology_terms(
                 tool["EDAM operation"], ontology=edam_ontology
             )
             tool["EDAM topic (no superclasses)"] = reduce_ontology_terms(tool["EDAM topic"], ontology=edam_ontology)
 
+            # add availability for all star servers
+            for name, url in USEGALAXY_STAR_SERVER_URLS.items():
```
Comment (bebatut) [inline bin/extract_galaxy_tools.py:735]:
```suggestion
            for name, url in USEGALAXY_SERVER_URLS.items():
```

File: bin/extract_galaxy_tools.py [hunk 3]
```diff
@@ -761,11 +724,17 @@ def reduce_ontology_terms(terms: List, ontology: Any) -> List:
...
+                tool[f"Tools available on {name}"] = check_tools_on_servers(tool["Galaxy tool ids"], url)
```
Comment (bebatut) [inline bin/extract_galaxy_tools.py:736]:
```suggestion
                tool[f"Available on {name}"] = check_tools_on_servers(tool["Galaxy tool ids"], url)
```

File: bin/extract_galaxy_tools.py [hunk 4]
```diff
@@ -761,11 +724,17 @@ def reduce_ontology_terms(terms: List, ontology: Any) -> List:
...
 
         for tool in tools:
+
+            # add EDAM terms without superclass
             tool["EDAM operation (no superclasses)"] = reduce_ontology_terms(
                 tool["EDAM operation"], ontology=edam_ontology
             )
             tool["EDAM topic (no superclasses)"] = reduce_ontology_terms(tool["EDAM topic"], ontology=edam_ontology)
 
+            # add availability for all star servers
```
Comment (bebatut) [inline bin/extract_galaxy_tools.py:734]:
```suggestion
            # add availability for UseGalaxy servers
```

**Comment by paulzierep** [comment]
Test output: https://github.com/paulzierep/galaxy_tool_extractor/blob/main/results/test.list_tools.json

**Pull request merged**